logfire/_internal/integrations/llm_providers/usage.py

```
@@ -36,21 +51,62 @@ def get_usage_attributes(
+                cache_write_tokens=cache_write_tokens,
             )
+            for candidate_id, candidate_url in candidates:
+                try:
+                    result['operation.cost'] = float(
+                        calc_price(
+                            price_usage, model_ref=model_ref, provider_id=candidate_id, provider_api_url=candidate_url
+                        ).total_price
+                    )
+                    break
+                except Exception:
+                    pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)